azure_monitor/setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = opentelemetry-azure-monitor-exporters-python
+description = Azure Monitor exporters using OpenTelemetry
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Microsoft
+author_email = appinsightssdk@microsoft.com
+url = https://github.com/microsoft/opentelemetry-azure-monitor-exporters-python
+platforms = any
+license = Apache-2.0
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+
+[options]
+python_requires = >=3.4
+package_dir=
+    =src
+packages=find_namespace:
+install_requires =
+    opentelemetry-api
+    opentelemetry-sdk
+
+[options.packages.find]
+where = src
```

Comment:
This is against github rule.

Reply:
Can you explain what you mean?

Reply:
![image](https://user-images.githubusercontent.com/17327289/67518048-c853c980-f658-11e9-9c86-b0d5859f36bf.png)